site/src/_pages/docs/0_index.mdx

```diff
@@ -1,12 +1,36 @@
 ## Overview
 
-These pages provide a practical introduction to implementing the block protocol, whether you are:
+The Block Protocol is a global standard for building interactive blocks connected to structured data.
 
-- interested in [developing blocks](https://blockprotocol.org/docs/developing-blocks), or
-- interested in [embedding blocks in your application](https://blockprotocol.org/docs/embedding-blocks)
+The following pages provide a practical introduction for developers who want to:
+
+- [develop blocks](https://blockprotocol.org/docs/developing-blocks) and publish them to the block registry
+- [embed blocks](https://blockprotocol.org/docs/embedding-blocks) in their own application
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### What is a block?
+
+Blocks are the basic units we use to build pages on the modern web. They are rectangular sections that contain a set of features and functions. They are sometimes also called _components_ or _widgets_.
+
+Each block has a type that determines the kind of data it can display and edit. You're probably familiar with the most common block types such as _paragraph_, _header_, _image_, _list_, and _table_. Some block types have more advanced interactive features such as filtering, search, and sorting.
+
+There are an infinite number of possible block types. One goal of the Block Protocol is to encourage developers to build and share brand new types of blocks that haven't been seen before, and for applications to offer users far more blocks than any one team could imagine or build.
+
+Blocks can be added to webpages using plain HTML and/or JavaScript. Some modern content management systems (eg. Wordpress, Squarespace) and applications (eg. Notion, Coda) allow you to add blocks to a page using a visual interface. With these block-based editors you can usually rearrange the order of blocks on a page using drag and drop. Blocks can be stacked on top of one another, arranged in columns, or embedded inside other blocks.
+
+From a developer perspective, blocks encapsulate a set of specific functionality. Building a block is no different to writing a React, Vue, or [Web Component](https://en.wikipedia.org/wiki/Web_Components) – in fact, these are all ways you might implement a block. They are reusable elements that expect certain data to be passed to them (what we refer to as a block's “schema”). For example, an image block might only accept an image URL in its `src` property, not a number.
```

Comment:
```suggestion
From a developer perspective, blocks encapsulate a set of specific functionality. Building a block is no different to writing a React, Vue, or [Web Component](https://en.wikipedia.org/wiki/Web_Components) – in fact, these are all ways you might implement a block. They are reusable elements that expect certain data to be passed to them (what we refer to as a block's ‘schema’). For example, an image block might only accept an image URL in its `src` property, not a number.
```

The most common rule I‘ve seen was: _“” for quotes, ‘’ for emphasis_. We don’t have to stick with it rigorously, but it might be worth keeping the rule in our heads.